content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,167 @@
+
+Federation is not enabled by default. You must create a cluster resource for the federation cluster and [register it](#register-a-single-cluster).
+
+Create, update, and delete clusters using sensuctl [create][5], [edit][6], and [delete][7] commands. Only cluster administrators can register a new cluster, but every user can [query the list of clusters][11].
+
+## What you can do with federation
+
+Federation affords visibility into the health of your infrastructure and services across multiple distinct Sensu instances within a single web UI. This is useful when you want to provide a single entry point for Sensu users who need to manage monitoring across multiple distinct physical data centers, cloud regions or providers.
+
+## JSON Web Tokens
+
+Federation uses JSON web tokens (JWTs) to authenticate Sensu users across different clusters using a single access token. JWTs are provided to users during authentication. Sensu uses asymmetric (public/private) keys to encrypt proof of the user's identity into the JWT payload. Therefore, federated clusters must share the same public and private JWT keys.
+
+## Step 1 Configure clusters with shared JWT keys
+
+JWTs support asymmetric encryption using RSA or Elliptic Curve keys. Use the `openssl` command line tool to generate a P-256 Elliptic Curve key:
+
+```shell
+openssl ecparam -genkey -name prime256v1 -noout -out ec_private.pem
+```
+
+and then generate a public key from the private key:
+
+```shell
+openssl ec -in ec_private.pem -pubout -out ec_public.pem
+```
+
+After generating these keys, copy them to your Sensu clusters. For this guide, we'll put JWT keys into `/etc/sensu/certs` and use the [`jwt-private-key-file` and `jwt-public-key-file` attributes][4] in `/etc/sensu/backend.yml` to specify the paths to these JWT keys:
+
+{{< highlight yml >}}
+jwt-private-key-file: /etc/sensu/certs/ec_private.pem
+jwt-public-key-file: /etc/sensu/certs/ec_public.pem
+{{< /highlight >}}
+
+After updating the backend configuration, restart `sensu-backend` for those settings to take effect.
+
+## Step 2 Create etcd replicators
+
+You can use etcd Replicators to synchronize [RBAC policy resources][10] between clusters. This allows you to centrally define permissions that replicate to all federated clusters, ensuring consistent access for Sensu users.
+
+Etcd replicators use the [etcd make-mirror utility][12] for one-way key replication. Our [etcd-replicators reference][2] includes [examples][9] for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources.
+
+## Step 3 Register clusters
+
+Each registered cluster must have a name and a list of cluster member URLs corresponding to the backend REST API.
+
+### Register a single cluster
+
+You do not need to register the cluster that you are currently operating from (for example, self or local-cluster) unless you want to configure the cluster name.
```

Comment:
Let's remove `self` here to avoid confusion.
```suggestion
You do not need to register the cluster that you are currently operating from (for example, local-cluster is automatically initialized) unless you want to configure the cluster name.
```